docs/design/shifu-framework/telemetry-service-SQLServer_MySQL.md

```diff
@@ -0,0 +1,66 @@
+# Telemetry Service SQL Endpoint Design
+
+## Introduction
+Telemetry Service is a standalone service that takes telemetry data collected by `deviceShifu` and fan-out it to designated endpoints for future process.
+
+## Design-Goal
+- Let telemetry service support pushing telemetries to SQL endpoints.
+
+## Design Non-Goal
+- Let telemetry service support any random endpoints
+- Let telemetry service serve as SQL service.
+
+## Design Details
+We will predefine the schema for the database, since the uncertainty of schema and data format will make the the feature hard to use and debug.
+
+The predefined schema looks like:
+```SQL
+CREATE TABLE DeviceMeasurements
```

Comment:
use a more generic name? i.e. Shifu Telemetry or let user configure the table name